Mathlib/Tactic/Basic.lean

```diff
@@ -193,8 +193,8 @@ macro_rules
   | `(tactic| by_contra $e) => `(tactic| (apply Classical.byContradiction; intro $e))
 
 /--
-`iterate n { ... }` runs the tactic block exactly `n` times.
-`iterate { ... }` runs the tactic block repeatedly until failure.
+`iterate n ( ... )` runs the tactic block exactly `n` times.
+`iterate ( ... )` runs the tactic block repeatedly until failure.
```

Comment:
```suggestion
`iterate n tac` runs `tac` exactly `n` times.
`iterate tac` runs `tac` repeatedly until failure.
```
No need to imply extra syntax, it just takes a tactic and `( ... )` is one way to make a tactic out of a sequence. Arguably the comment should also mention that
```
iterate
  tac1
  tac2
```
also works; most tactic combinators take a `tacticSeq` so you can use newline-separated tactic blocks.